detekt-config.yml

```diff
@@ -448,7 +451,8 @@ style:
     active: false
     imports: ''
   ForbiddenVoid:
-    active: false
+    #LeakCanary - enabled
+    active: true
```

Comment:
What's that about?

Reply:
Would be nice if commit message + PR description outlined what the enabled rules are about.

Reply:
Reports usage of `Void` where there should be `Unit` instead
https://arturbosch.github.io/detekt/style.html#forbiddenvoid